src/apps/reservation-list/modal/reservation-details/helper.ts

```diff
@@ -0,0 +1,34 @@
+import { ComplexSearchWithPaginationWorkAccessQuery } from "../../../../core/dbc-gateway/generated/graphql";
+
+type AccessManifestations =
+  ComplexSearchWithPaginationWorkAccessQuery["complexSearch"]["works"][0]["manifestations"]["all"];
+
+export const findAccessManifestationByIdentifier = (
+  manifestations: AccessManifestations,
+  identifier: string
+) => {
+  return manifestations.filter((manifestation) =>
```

Comment:
Since you do not use curly brackets in the anonymous function in `filter`, you might as well skip them + `return` here.